misc/librenms-irc.service

```diff
@@ -0,0 +1,12 @@
+# /etc/systemd/system/librenms-irc.service 
+
+librenms-irc.service
+[Unit]
+Description=Librenms IRC bot
+
+[Service]
+ExecStart=/opt/librenms/irc.php
```

Comment:
@murrant I think it's worth doing before this is merged in. It only means people have to re-deploy this when it is updated which isn't part of our updates. I know it's not _needed_ but it would be good to start it right.